hash.js

```diff
@@ -9,11 +7,6 @@ function Hash (blockSize, finalSize) {
   this._s = 0
 }
 
-Hash.prototype.init = function () {
```

Comment:
Removed because these variables are only ever initialized once, and they are done so in the constructor.